src/components/SearchResults.js

```diff
@@ -0,0 +1,22 @@
+import React, { useState, useEffect } from "react";
+import TableInfo from "./TableInfo.js";
+import TableData from "./TableData.js";
+
+function SearchResults() {
+  const [bookings, setBookings] = useState([]);
+  useEffect(() => {
+    fetch("https://cyf-react.glitch.me.")
+      .then(response => response.json())
+      .then(data => setBookings(data));
+  }, []);
```

Comment:
Great use of React hooks and fetch method!👏 But according to instructions, data should be fetched in the `Bookings` component and passed to `SearchResults` as props.